src/tests/ScoreBoard.spec.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { render, cleanup } from 'react-testing-library';
+import { ScoreBoard } from '../ScoreBoard';
+
+const originalLocalStorage = global.window;
+
+const mockLocalStorage = {
+  getItem: function (key) {
+    if (key === 'userMoves') {
+      return JSON.stringify([
+        { name: 'John', moves: 20 },
+        { name: 'Andy', moves: 8 },
+        { name: 'Bill', moves: 10 },
+      ])
+    }
+  },
+  setItem: jest.fn(),
+  clear: jest.fn()
+}
+
+beforeAll(() => {
+  global.localStorage = mockLocalStorage;
+})
+
+describe('ScoreBoard component', () => {
+  it('Should render 3 list if three items is stored on local storage', () => {
+    const { container } = render(<ScoreBoard />);
+    expect(container.querySelectorAll('li')).toHaveLength(3)
+  }) 
+
+  it('Should render 3 list correctly', () => {
+    const { container } = render(<ScoreBoard />);
+    expect(container.querySelectorAll('li')[0].querySelector('strong').innerHTML).toEqual('Andy')
```

Comment:
@devTeaa take a look at this comment https://github.com/itaditya/trick-or-treat-game/pull/57#discussion_r331752918. You might have missed it.

If it doesn't work, the do this

expect(container.querySelectorAll('li')[0].innerText).toEqual('Andy: 8 moves');